static/styles/settings.css

```diff
@@ -569,9 +573,28 @@ input[type="checkbox"] {
     margin-top: 10px;
 }
 
-.language_selection_widget .language_selection_button {
-    text-decoration: none;
-    margin-left: 20px;
+.language_selection_widget {
+    margin-bottom: 20px;
```

Comment:
I removed it because, it wasn't working well here.

Reply:
Ok, for it to work with botht the widgets, we need to add the class to `.language_selection_widget` element, but other than that I am not sure why this wouldn't work since it applies exact same CSS.

Reply:
Ok, I changed it, since I don't recall the reason why I removed it. It may have to do with `input-group` being associated with dropdowns.